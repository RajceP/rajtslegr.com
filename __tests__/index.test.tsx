import { getPage } from 'next-page-tester';
import prettier from 'prettier';

describe('Index page', () => {
  it('should match the snapshot', async () => {
    const { serverRenderToString, render } = await getPage({
      route: '/',
      useDocument: true,
    });

    // check correctness of SSR result
    const { html } = serverRenderToString();
    const formattedSSRSnapshot = prettier.format(html, {
      parser: 'html',
    });

    expect(formattedSSRSnapshot).toMatchSnapshot();

    // check hydrated app
    const { nextRoot } = render();
    const formattedHydratedSnapshot = prettier.format(nextRoot.outerHTML, {
      parser: 'html',
    });

    expect(formattedHydratedSnapshot).toMatchSnapshot();
    expect(formattedHydratedSnapshot).toContain('<img>');
  });
});
