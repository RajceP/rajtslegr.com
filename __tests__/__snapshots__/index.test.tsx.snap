// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index page should match the snapshot 1`] = `
"<html lang=\\"en\\">
  <head>
    <script
      async=\\"\\"
      src=\\"https://www.googletagmanager.com/gtag/js?id=undefined\\"
    ></script>
    <script>
      window.dataLayer = window.dataLayer || [];
      function gtag() {
        dataLayer.push(arguments);
      }
      gtag(\\"js\\", new Date());
      gtag(\\"config\\", \\"undefined\\", {
        page_path: window.location.pathname,
      });
    </script>
    <link rel=\\"preconnect\\" href=\\"https://www.google-analytics.com\\" />
    <link rel=\\"shortcut icon\\" type=\\"image/jpg\\" href=\\"/static/images/hero.png\\" />
    <meta property=\\"title\\" content=\\"Petr Rajtslegr | Full Stack Dev\\" />
    <meta
      name=\\"description\\"
      content=\\"Petr Rajtslegr - Full Stack developer based in Prague, Czech Republic.\\"
    />
    <meta
      name=\\"keywords\\"
      content=\\"HTML, CSS, JavaScript, developer, Petr, Rajtslegr\\"
    />
    <meta property=\\"og:type\\" content=\\"website\\" />
    <meta property=\\"og:url\\" content=\\"https://rajtslegr.com/\\" />
    <meta property=\\"og:title\\" content=\\"Petr Rajtslegr | Full Stack Dev\\" />
    <meta
      property=\\"og:description\\"
      content=\\"Petr Rajtslegr - Full Stack developer based in Prague, Czech Republic.\\"
    />
    <meta name=\\"next-head-count\\" content=\\"0\\" />
    <noscript data-n-css=\\"\\"></noscript>
  </head>
  <body>
    <div id=\\"__next\\">
      <div class=\\"flex flex-col items-center min-h-screen\\">
        <nav
          class=\\"fixed w-full bg-white dark:bg-dark z-10 null transition-shadow\\"
        >
          <div
            class=\\"
              container
              flex
              items-center
              justify-between
              h-full
              px-4
              mx-auto
              xl:w-4/6
            \\"
          >
            <ul>
              <li class=\\"space-x-2\\">
                <a
                  class=\\"
                    text-gray-500
                    transition
                    dark:text-gray-400
                    hover:text-gray-600
                    dark:hover:text-gray-300
                  \\"
                  href=\\"/\\"
                  >Home</a
                ><a
                  class=\\"
                    text-gray-500
                    transition
                    dark:text-gray-400
                    hover:text-gray-600
                    dark:hover:text-gray-300
                  \\"
                  href=\\"/dashboard\\"
                  >Dashboard</a
                ><a
                  class=\\"
                    text-gray-500
                    transition
                    dark:text-gray-400
                    hover:text-gray-600
                    dark:hover:text-gray-300
                  \\"
                  href=\\"/projects\\"
                  >Projects</a
                ><a
                  class=\\"
                    text-gray-500
                    transition
                    dark:text-gray-400
                    hover:text-gray-600
                    dark:hover:text-gray-300
                  \\"
                  href=\\"/blog\\"
                  >Blog</a
                ><a
                  class=\\"
                    text-gray-500
                    transition
                    dark:text-gray-400
                    hover:text-gray-600
                    dark:hover:text-gray-300
                  \\"
                  href=\\"/contact\\"
                  >Contact</a
                >
              </li>
            </ul>
            <button
              aria-label=\\"Toggle Dark Mode\\"
              type=\\"button\\"
              class=\\"
                w-12
                h-12
                p-3
                my-3
                transition
                bg-gray-200
                rounded-lg
                shadow
                dark:bg-gray-600
                hover:shadow-lg
              \\"
            ></button>
          </div>
        </nav>
        <div class=\\"flex-auto w-full pt-16 md:pt-24\\">
          <main
            class=\\"container px-4 py-6 mx-auto my-4 md:py-12 xl:w-5/6 2xl:w-2/3\\"
          >
            <div
              class=\\"
                flex flex-col-reverse
                items-center
                justify-between
                xl:justify-around
                gap-y-8
                lg:flex-row
              \\"
            >
              <div
                class=\\"
                  flex flex-col
                  w-full
                  p-4
                  overflow-auto
                  text-sm
                  transition
                  border border-gray-200
                  rounded
                  shadow
                  font-monos
                  lg:max-w-lg
                \\"
              >
                <pre
                  class=\\"flex flex-row gap-6 break-all whitespace-pre-wrap\\"
                ><span class=\\"leading-relaxed text-gray-500 select-none dark:text-gray-400\\"> 1</span><span class=\\"text-gray-500 dark:text-gray-400\\">// About me</span></pre>
                <pre
                  class=\\"flex flex-row gap-6 break-all whitespace-pre-wrap\\"
                ><span class=\\"leading-relaxed text-gray-500 select-none dark:text-gray-400\\"> 2</span><span class=\\"\\">const name = &#x27;Petr Rajtslegr&#x27;;</span></pre>
                <pre
                  class=\\"flex flex-row gap-6 break-all whitespace-pre-wrap\\"
                ><span class=\\"leading-relaxed text-gray-500 select-none dark:text-gray-400\\"> 3</span><span class=\\"\\">const profession = &#x27;Full Stack Developer&#x27;;</span></pre>
                <pre
                  class=\\"flex flex-row gap-6 break-all whitespace-pre-wrap\\"
                ><span class=\\"leading-relaxed text-gray-500 select-none dark:text-gray-400\\"> 4</span><span class=\\"\\">const location = &#x27;Prague, Czech Republic&#x27;;</span></pre>
                <pre
                  class=\\"flex flex-row gap-6 break-all whitespace-pre-wrap\\"
                ><span class=\\"leading-relaxed text-gray-500 select-none dark:text-gray-400\\"> 5</span><span class=\\"\\"></span></pre>
                <pre
                  class=\\"flex flex-row gap-6 break-all whitespace-pre-wrap\\"
                ><span class=\\"leading-relaxed text-gray-500 select-none dark:text-gray-400\\"> 6</span><span class=\\"text-gray-500 dark:text-gray-400\\">// What I do</span></pre>
                <pre
                  class=\\"flex flex-row gap-6 break-all whitespace-pre-wrap\\"
                ><span class=\\"leading-relaxed text-gray-500 select-none dark:text-gray-400\\"> 7</span><span class=\\"\\">const languages = [</span></pre>
                <pre
                  class=\\"flex flex-row gap-6 break-all whitespace-pre-wrap\\"
                ><span class=\\"leading-relaxed text-gray-500 select-none dark:text-gray-400\\"> 8</span><span class=\\"\\">  &#x27;JavaScript&#x27;,</span></pre>
                <pre
                  class=\\"flex flex-row gap-6 break-all whitespace-pre-wrap\\"
                ><span class=\\"leading-relaxed text-gray-500 select-none dark:text-gray-400\\"> 9</span><span class=\\"\\">  &#x27;TypeScript&#x27;</span></pre>
                <pre
                  class=\\"flex flex-row gap-6 break-all whitespace-pre-wrap\\"
                ><span class=\\"leading-relaxed text-gray-500 select-none dark:text-gray-400\\">10</span><span class=\\"\\">];</span></pre>
                <pre
                  class=\\"flex flex-row gap-6 break-all whitespace-pre-wrap\\"
                ><span class=\\"leading-relaxed text-gray-500 select-none dark:text-gray-400\\">11</span><span class=\\"\\"></span></pre>
                <pre
                  class=\\"flex flex-row gap-6 break-all whitespace-pre-wrap\\"
                ><span class=\\"leading-relaxed text-gray-500 select-none dark:text-gray-400\\">12</span><span class=\\"\\">const technologies = [</span></pre>
                <pre
                  class=\\"flex flex-row gap-6 break-all whitespace-pre-wrap\\"
                ><span class=\\"leading-relaxed text-gray-500 select-none dark:text-gray-400\\">13</span><span class=\\"\\">  &#x27;React&#x27;,</span></pre>
                <pre
                  class=\\"flex flex-row gap-6 break-all whitespace-pre-wrap\\"
                ><span class=\\"leading-relaxed text-gray-500 select-none dark:text-gray-400\\">14</span><span class=\\"\\">  &#x27;Angular&#x27;,</span></pre>
                <pre
                  class=\\"flex flex-row gap-6 break-all whitespace-pre-wrap\\"
                ><span class=\\"leading-relaxed text-gray-500 select-none dark:text-gray-400\\">15</span><span class=\\"\\">  &#x27;Next.js&#x27;,</span></pre>
                <pre
                  class=\\"flex flex-row gap-6 break-all whitespace-pre-wrap\\"
                ><span class=\\"leading-relaxed text-gray-500 select-none dark:text-gray-400\\">16</span><span class=\\"\\">  &#x27;Node.js&#x27;</span></pre>
                <pre
                  class=\\"flex flex-row gap-6 break-all whitespace-pre-wrap\\"
                ><span class=\\"leading-relaxed text-gray-500 select-none dark:text-gray-400\\">17</span><span class=\\"\\">];</span></pre>
              </div>
              <div
                style=\\"
                  display: inline-block;
                  max-width: 100%;
                  overflow: hidden;
                  position: relative;
                  box-sizing: border-box;
                  margin: 0;
                \\"
              >
                <div
                  style=\\"
                    box-sizing: border-box;
                    display: block;
                    max-width: 100%;
                  \\"
                >
                  <img
                    style=\\"
                      max-width: 100%;
                      display: block;
                      margin: 0;
                      border: none;
                      padding: 0;
                    \\"
                    alt=\\"\\"
                    aria-hidden=\\"true\\"
                    role=\\"presentation\\"
                    src=\\"data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMzQ0IiBoZWlnaHQ9IjM0NCIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIiB2ZXJzaW9uPSIxLjEiLz4=\\"
                  />
                </div>
                <img
                  alt=\\"Hero\\"
                  srcset=\\"
                    /_next/image?url=%2Fstatic%2Fimages%2Fhero.png&amp;w=384&amp;q=75 1x,
                    /_next/image?url=%2Fstatic%2Fimages%2Fhero.png&amp;w=750&amp;q=75 2x
                  \\"
                  src=\\"/_next/image?url=%2Fstatic%2Fimages%2Fhero.png&amp;w=750&amp;q=75\\"
                  decoding=\\"async\\"
                  class=\\"rounded-full shadow\\"
                  style=\\"
                    position: absolute;
                    top: 0;
                    left: 0;
                    bottom: 0;
                    right: 0;
                    box-sizing: border-box;
                    padding: 0;
                    border: none;
                    margin: auto;
                    display: block;
                    width: 0;
                    height: 0;
                    min-width: 100%;
                    max-width: 100%;
                    min-height: 100%;
                    max-height: 100%;
                  \\"
                />
              </div>
            </div>
          </main>
        </div>
        <div class=\\"flex flex-col items-center w-full p-2 space-y-12\\">
          <div
            class=\\"
              flex flex-col
              w-11/12
              border-t border-gray-200
              sm:w-3/4
              sm:p-6
              md:w-1/2
            \\"
          >
            <div class=\\"flex flex-row justify-between md:justify-evenly\\">
              <div class=\\"flex flex-col space-y-2\\">
                <a
                  class=\\"
                    text-gray-500
                    transition
                    dark:text-gray-400
                    hover:text-gray-600
                    dark:hover:text-gray-300
                  \\"
                  href=\\"/\\"
                  >Home</a
                ><a
                  class=\\"
                    text-gray-500
                    transition
                    dark:text-gray-400
                    hover:text-gray-600
                    dark:hover:text-gray-300
                  \\"
                  href=\\"/projects\\"
                  >Projects</a
                ><a
                  class=\\"
                    text-gray-500
                    transition
                    dark:text-gray-400
                    hover:text-gray-600
                    dark:hover:text-gray-300
                  \\"
                  href=\\"/blog\\"
                  >Blog</a
                ><a
                  class=\\"
                    text-gray-500
                    transition
                    dark:text-gray-400
                    hover:text-gray-600
                    dark:hover:text-gray-300
                  \\"
                  href=\\"/contact\\"
                  >Contact</a
                >
              </div>
              <div class=\\"flex flex-col space-y-2\\">
                <a
                  class=\\"
                    text-gray-500
                    transition
                    dark:text-gray-400
                    hover:text-gray-600
                    dark:hover:text-gray-300
                  \\"
                  target=\\"_blank\\"
                  rel=\\"noopener noreferrer\\"
                  href=\\"https://github.com/RajceP\\"
                  >GitHub</a
                ><a
                  class=\\"
                    text-gray-500
                    transition
                    dark:text-gray-400
                    hover:text-gray-600
                    dark:hover:text-gray-300
                  \\"
                  target=\\"_blank\\"
                  rel=\\"noopener noreferrer\\"
                  href=\\"https://www.linkedin.com/in/petr-rajtšlégr-5aa84a16b\\"
                  >LinkedIn</a
                ><a
                  class=\\"
                    text-gray-500
                    transition
                    dark:text-gray-400
                    hover:text-gray-600
                    dark:hover:text-gray-300
                  \\"
                  target=\\"_blank\\"
                  rel=\\"noopener noreferrer\\"
                  href=\\"https://www.facebook.com/petr.rajtslegr\\"
                  >Facebook</a
                ><a
                  class=\\"
                    text-gray-500
                    transition
                    dark:text-gray-400
                    hover:text-gray-600
                    dark:hover:text-gray-300
                  \\"
                  target=\\"_blank\\"
                  rel=\\"noopener noreferrer\\"
                  href=\\"https://twitter.com/RajceP\\"
                  >Twitter</a
                >
              </div>
              <div class=\\"flex flex-col space-y-2\\">
                <a
                  class=\\"
                    text-gray-500
                    transition
                    dark:text-gray-400
                    hover:text-gray-600
                    dark:hover:text-gray-300
                  \\"
                  target=\\"_blank\\"
                  rel=\\"noopener noreferrer\\"
                  href=\\"https://www.instagram.com/rajce\\"
                  >Instagram</a
                ><a
                  class=\\"
                    text-gray-500
                    transition
                    dark:text-gray-400
                    hover:text-gray-600
                    dark:hover:text-gray-300
                  \\"
                  target=\\"_blank\\"
                  rel=\\"noopener noreferrer\\"
                  href=\\"https://t.me/rajcep\\"
                  >Telegram</a
                ><a
                  class=\\"
                    text-gray-500
                    transition
                    dark:text-gray-400
                    hover:text-gray-600
                    dark:hover:text-gray-300
                  \\"
                  target=\\"_blank\\"
                  rel=\\"noopener noreferrer\\"
                  href=\\"https://www.last.fm/user/RajceP\\"
                  >Last.fm</a
                ><a
                  class=\\"
                    text-gray-500
                    transition
                    dark:text-gray-400
                    hover:text-gray-600
                    dark:hover:text-gray-300
                  \\"
                  target=\\"_blank\\"
                  rel=\\"noopener noreferrer\\"
                  href=\\"https://soundcloud.com/rajcep\\"
                  >SoundCloud</a
                >
              </div>
            </div>
          </div>
          <div class=\\"flex flex-row justify-center w-full\\">
            <div
              class=\\"
                flex flex-row
                items-center
                text-gray-500
                dark:text-gray-400
                hover:text-gray-600
                dark:hover:text-gray-300
              \\"
            >
              Made by Petr Rajtslegr with <!-- --> and<!-- --> © <!-- -->2021
            </div>
          </div>
        </div>
      </div>
    </div>
    <script id=\\"__NEXT_DATA__\\" type=\\"application/json\\">
      {
        \\"page\\": \\"/index\\",
        \\"query\\": {},
        \\"buildId\\": \\"next-page-tester\\",
        \\"props\\": {
          \\"pageProps\\": {
            \\"heroCode\\": [
              \\"// About me\\",
              \\"const name = 'Petr Rajtslegr';\\",
              \\"const profession = 'Full Stack Developer';\\",
              \\"const location = 'Prague, Czech Republic';\\",
              \\"\\",
              \\"// What I do\\",
              \\"const languages = [\\",
              \\"  'JavaScript',\\",
              \\"  'TypeScript'\\",
              \\"];\\",
              \\"\\",
              \\"const technologies = [\\",
              \\"  'React',\\",
              \\"  'Angular',\\",
              \\"  'Next.js',\\",
              \\"  'Node.js'\\",
              \\"];\\"
            ]
          }
        }
      }
    </script>
  </body>
</html>
"
`;

exports[`Index page should match the snapshot 2`] = `
"<div id=\\"__next\\">
  <div class=\\"flex flex-col items-center min-h-screen\\">
    <nav class=\\"fixed w-full bg-white dark:bg-dark z-10 null transition-shadow\\">
      <div
        class=\\"
          container
          flex
          items-center
          justify-between
          h-full
          px-4
          mx-auto
          xl:w-4/6
        \\"
      >
        <ul>
          <li class=\\"space-x-2\\">
            <a
              class=\\"
                text-gray-500
                transition
                dark:text-gray-400
                hover:text-gray-600
                dark:hover:text-gray-300
              \\"
              href=\\"/\\"
              >Home</a
            ><a
              class=\\"
                text-gray-500
                transition
                dark:text-gray-400
                hover:text-gray-600
                dark:hover:text-gray-300
              \\"
              href=\\"/dashboard\\"
              >Dashboard</a
            ><a
              class=\\"
                text-gray-500
                transition
                dark:text-gray-400
                hover:text-gray-600
                dark:hover:text-gray-300
              \\"
              href=\\"/projects\\"
              >Projects</a
            ><a
              class=\\"
                text-gray-500
                transition
                dark:text-gray-400
                hover:text-gray-600
                dark:hover:text-gray-300
              \\"
              href=\\"/blog\\"
              >Blog</a
            ><a
              class=\\"
                text-gray-500
                transition
                dark:text-gray-400
                hover:text-gray-600
                dark:hover:text-gray-300
              \\"
              href=\\"/contact\\"
              >Contact</a
            >
          </li>
        </ul>
        <button
          aria-label=\\"Toggle Dark Mode\\"
          type=\\"button\\"
          class=\\"
            w-12
            h-12
            p-3
            my-3
            transition
            bg-gray-200
            rounded-lg
            shadow
            dark:bg-gray-600
            hover:shadow-lg
          \\"
        ></button>
      </div>
    </nav>
    <div class=\\"flex-auto w-full pt-16 md:pt-24\\">
      <main
        class=\\"container px-4 py-6 mx-auto my-4 md:py-12 xl:w-5/6 2xl:w-2/3\\"
      >
        <div
          class=\\"
            flex flex-col-reverse
            items-center
            justify-between
            xl:justify-around
            gap-y-8
            lg:flex-row
          \\"
        >
          <div
            class=\\"
              flex flex-col
              w-full
              p-4
              overflow-auto
              text-sm
              transition
              border border-gray-200
              rounded
              shadow
              font-monos
              lg:max-w-lg
            \\"
          >
            <pre
              class=\\"flex flex-row gap-6 break-all whitespace-pre-wrap\\"
            ><span class=\\"leading-relaxed text-gray-500 select-none dark:text-gray-400\\"> 1</span><span class=\\"text-gray-500 dark:text-gray-400\\">// About me</span></pre>
            <pre
              class=\\"flex flex-row gap-6 break-all whitespace-pre-wrap\\"
            ><span class=\\"leading-relaxed text-gray-500 select-none dark:text-gray-400\\"> 2</span><span class=\\"\\">const name = 'Petr Rajtslegr';</span></pre>
            <pre
              class=\\"flex flex-row gap-6 break-all whitespace-pre-wrap\\"
            ><span class=\\"leading-relaxed text-gray-500 select-none dark:text-gray-400\\"> 3</span><span class=\\"\\">const profession = 'Full Stack Developer';</span></pre>
            <pre
              class=\\"flex flex-row gap-6 break-all whitespace-pre-wrap\\"
            ><span class=\\"leading-relaxed text-gray-500 select-none dark:text-gray-400\\"> 4</span><span class=\\"\\">const location = 'Prague, Czech Republic';</span></pre>
            <pre
              class=\\"flex flex-row gap-6 break-all whitespace-pre-wrap\\"
            ><span class=\\"leading-relaxed text-gray-500 select-none dark:text-gray-400\\"> 5</span><span class=\\"\\"></span></pre>
            <pre
              class=\\"flex flex-row gap-6 break-all whitespace-pre-wrap\\"
            ><span class=\\"leading-relaxed text-gray-500 select-none dark:text-gray-400\\"> 6</span><span class=\\"text-gray-500 dark:text-gray-400\\">// What I do</span></pre>
            <pre
              class=\\"flex flex-row gap-6 break-all whitespace-pre-wrap\\"
            ><span class=\\"leading-relaxed text-gray-500 select-none dark:text-gray-400\\"> 7</span><span class=\\"\\">const languages = [</span></pre>
            <pre
              class=\\"flex flex-row gap-6 break-all whitespace-pre-wrap\\"
            ><span class=\\"leading-relaxed text-gray-500 select-none dark:text-gray-400\\"> 8</span><span class=\\"\\">  'JavaScript',</span></pre>
            <pre
              class=\\"flex flex-row gap-6 break-all whitespace-pre-wrap\\"
            ><span class=\\"leading-relaxed text-gray-500 select-none dark:text-gray-400\\"> 9</span><span class=\\"\\">  'TypeScript'</span></pre>
            <pre
              class=\\"flex flex-row gap-6 break-all whitespace-pre-wrap\\"
            ><span class=\\"leading-relaxed text-gray-500 select-none dark:text-gray-400\\">10</span><span class=\\"\\">];</span></pre>
            <pre
              class=\\"flex flex-row gap-6 break-all whitespace-pre-wrap\\"
            ><span class=\\"leading-relaxed text-gray-500 select-none dark:text-gray-400\\">11</span><span class=\\"\\"></span></pre>
            <pre
              class=\\"flex flex-row gap-6 break-all whitespace-pre-wrap\\"
            ><span class=\\"leading-relaxed text-gray-500 select-none dark:text-gray-400\\">12</span><span class=\\"\\">const technologies = [</span></pre>
            <pre
              class=\\"flex flex-row gap-6 break-all whitespace-pre-wrap\\"
            ><span class=\\"leading-relaxed text-gray-500 select-none dark:text-gray-400\\">13</span><span class=\\"\\">  'React',</span></pre>
            <pre
              class=\\"flex flex-row gap-6 break-all whitespace-pre-wrap\\"
            ><span class=\\"leading-relaxed text-gray-500 select-none dark:text-gray-400\\">14</span><span class=\\"\\">  'Angular',</span></pre>
            <pre
              class=\\"flex flex-row gap-6 break-all whitespace-pre-wrap\\"
            ><span class=\\"leading-relaxed text-gray-500 select-none dark:text-gray-400\\">15</span><span class=\\"\\">  'Next.js',</span></pre>
            <pre
              class=\\"flex flex-row gap-6 break-all whitespace-pre-wrap\\"
            ><span class=\\"leading-relaxed text-gray-500 select-none dark:text-gray-400\\">16</span><span class=\\"\\">  'Node.js'</span></pre>
            <pre
              class=\\"flex flex-row gap-6 break-all whitespace-pre-wrap\\"
            ><span class=\\"leading-relaxed text-gray-500 select-none dark:text-gray-400\\">17</span><span class=\\"\\">];</span></pre>
          </div>
          <div
            style=\\"
              display: inline-block;
              max-width: 100%;
              overflow: hidden;
              position: relative;
              box-sizing: border-box;
              margin: 0;
            \\"
          >
            <div
              style=\\"box-sizing: border-box; display: block; max-width: 100%\\"
            >
              <img
                style=\\"
                  max-width: 100%;
                  display: block;
                  margin: 0;
                  border: none;
                  padding: 0;
                \\"
                alt=\\"\\"
                aria-hidden=\\"true\\"
                role=\\"presentation\\"
                src=\\"data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMzQ0IiBoZWlnaHQ9IjM0NCIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIiB2ZXJzaW9uPSIxLjEiLz4=\\"
              />
            </div>
            <img
              alt=\\"Hero\\"
              srcset=\\"
                /_next/image?url=%2Fstatic%2Fimages%2Fhero.png&amp;w=384&amp;q=75 1x,
                /_next/image?url=%2Fstatic%2Fimages%2Fhero.png&amp;w=750&amp;q=75 2x
              \\"
              src=\\"/_next/image?url=%2Fstatic%2Fimages%2Fhero.png&amp;w=750&amp;q=75\\"
              decoding=\\"async\\"
              class=\\"rounded-full shadow\\"
              style=\\"
                position: absolute;
                top: 0;
                left: 0;
                bottom: 0;
                right: 0;
                box-sizing: border-box;
                padding: 0;
                border: none;
                margin: auto;
                display: block;
                width: 0;
                height: 0;
                min-width: 100%;
                max-width: 100%;
                min-height: 100%;
                max-height: 100%;
              \\"
            />
          </div>
        </div>
      </main>
    </div>
    <div class=\\"flex flex-col items-center w-full p-2 space-y-12\\">
      <div
        class=\\"
          flex flex-col
          w-11/12
          border-t border-gray-200
          sm:w-3/4
          sm:p-6
          md:w-1/2
        \\"
      >
        <div class=\\"flex flex-row justify-between md:justify-evenly\\">
          <div class=\\"flex flex-col space-y-2\\">
            <a
              class=\\"
                text-gray-500
                transition
                dark:text-gray-400
                hover:text-gray-600
                dark:hover:text-gray-300
              \\"
              href=\\"/\\"
              >Home</a
            ><a
              class=\\"
                text-gray-500
                transition
                dark:text-gray-400
                hover:text-gray-600
                dark:hover:text-gray-300
              \\"
              href=\\"/projects\\"
              >Projects</a
            ><a
              class=\\"
                text-gray-500
                transition
                dark:text-gray-400
                hover:text-gray-600
                dark:hover:text-gray-300
              \\"
              href=\\"/blog\\"
              >Blog</a
            ><a
              class=\\"
                text-gray-500
                transition
                dark:text-gray-400
                hover:text-gray-600
                dark:hover:text-gray-300
              \\"
              href=\\"/contact\\"
              >Contact</a
            >
          </div>
          <div class=\\"flex flex-col space-y-2\\">
            <a
              class=\\"
                text-gray-500
                transition
                dark:text-gray-400
                hover:text-gray-600
                dark:hover:text-gray-300
              \\"
              target=\\"_blank\\"
              rel=\\"noopener noreferrer\\"
              href=\\"https://github.com/RajceP\\"
              >GitHub</a
            ><a
              class=\\"
                text-gray-500
                transition
                dark:text-gray-400
                hover:text-gray-600
                dark:hover:text-gray-300
              \\"
              target=\\"_blank\\"
              rel=\\"noopener noreferrer\\"
              href=\\"https://www.linkedin.com/in/petr-rajtšlégr-5aa84a16b\\"
              >LinkedIn</a
            ><a
              class=\\"
                text-gray-500
                transition
                dark:text-gray-400
                hover:text-gray-600
                dark:hover:text-gray-300
              \\"
              target=\\"_blank\\"
              rel=\\"noopener noreferrer\\"
              href=\\"https://www.facebook.com/petr.rajtslegr\\"
              >Facebook</a
            ><a
              class=\\"
                text-gray-500
                transition
                dark:text-gray-400
                hover:text-gray-600
                dark:hover:text-gray-300
              \\"
              target=\\"_blank\\"
              rel=\\"noopener noreferrer\\"
              href=\\"https://twitter.com/RajceP\\"
              >Twitter</a
            >
          </div>
          <div class=\\"flex flex-col space-y-2\\">
            <a
              class=\\"
                text-gray-500
                transition
                dark:text-gray-400
                hover:text-gray-600
                dark:hover:text-gray-300
              \\"
              target=\\"_blank\\"
              rel=\\"noopener noreferrer\\"
              href=\\"https://www.instagram.com/rajce\\"
              >Instagram</a
            ><a
              class=\\"
                text-gray-500
                transition
                dark:text-gray-400
                hover:text-gray-600
                dark:hover:text-gray-300
              \\"
              target=\\"_blank\\"
              rel=\\"noopener noreferrer\\"
              href=\\"https://t.me/rajcep\\"
              >Telegram</a
            ><a
              class=\\"
                text-gray-500
                transition
                dark:text-gray-400
                hover:text-gray-600
                dark:hover:text-gray-300
              \\"
              target=\\"_blank\\"
              rel=\\"noopener noreferrer\\"
              href=\\"https://www.last.fm/user/RajceP\\"
              >Last.fm</a
            ><a
              class=\\"
                text-gray-500
                transition
                dark:text-gray-400
                hover:text-gray-600
                dark:hover:text-gray-300
              \\"
              target=\\"_blank\\"
              rel=\\"noopener noreferrer\\"
              href=\\"https://soundcloud.com/rajcep\\"
              >SoundCloud</a
            >
          </div>
        </div>
      </div>
      <div class=\\"flex flex-row justify-center w-full\\">
        <div
          class=\\"
            flex flex-row
            items-center
            text-gray-500
            dark:text-gray-400
            hover:text-gray-600
            dark:hover:text-gray-300
          \\"
        >
          Made by Petr Rajtslegr with&nbsp;<!-- -->&nbsp;and<!-- -->&nbsp;©&nbsp;<!-- -->2021
        </div>
      </div>
    </div>
  </div>
</div>
"
`;
